src/fragments/DisplayNotification.tsx

```diff
@@ -0,0 +1,79 @@
+import { isUserMessage, SystemSentMessageNotification } from 'chatkitty';
+import moment from 'moment';
+import React from 'react';
+import { FlexColumn, FlexRow, StyledBox } from 'react-chat-ui-kit';
+
+import UserAvatar from './UserAvatar';
+
+import './../styles/slide.css';
+
+interface notificationProp {
+  notification: SystemSentMessageNotification ;
+  isMentionNotification?: boolean | null;
+}
+
+const DisplayNotification: React.FC<notificationProp> = ({
+  notification,
+  isMentionNotification,
+}: notificationProp) => {
+  return (
+    <StyledBox
+      className="slideIn"
+      style={{
+        width: '250px',
+        height: '90px',
+        background: 'white',
+        position: 'absolute',
+        bottom: '5px',
+        left: '5px',
+        borderRadius: '5%',
+        cursor: 'pointer',
+      }}
+    >
+      {isUserMessage(notification.data.message) && (
```

Comment:
Consider using the `isUserMentionedNotification` helper type function instead.

Reply:
using `isUserMentionedNotification` will need additional unnecessary code